packages/at_chops/test/at_chops_test.dart

```diff
@@ -0,0 +1,49 @@
+import 'dart:convert';
+import 'dart:typed_data';
+
+import 'package:at_chops/src/algorithm/default_encryption_algo.dart';
+import 'package:at_chops/src/algorithm/default_signing_algo.dart';
+import 'package:at_chops/src/at_chops_impl.dart';
+import 'package:at_chops/src/key/aes_key.dart';
+import 'package:at_chops/src/util/at_chops_util.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('A group of tests for encryption and decryption', () {
+    test('Test symmetric encrypt/decrypt bytes with initialisation vector', () {
+      String data = 'Hello World';
+      final atChops = AtChopsImpl();
+      AESKey aesKey = AESKey(AtChopsUtil.generateAESKey(32));
+      final iv = AtChopsUtil.generateIV(16);
+      final algo = DefaultEncryptionAlgo(aesKey);
+      final encryptedBytes =
+          atChops.encryptBytes(utf8.encode(data) as Uint8List, algo, iv: iv);
+      final decryptedBytes = atChops.decryptBytes(encryptedBytes, algo, iv: iv);
+      expect(utf8.decode(decryptedBytes), data);
+    });
+    test('Test symmetric encrypt/decrypt string with initialisation vector',
```

Comment:
this test is failing currently..have to check whether utf8 or base64 encoding is appropriate

Reply:
fixed the issue